Vagrantfile

```diff
@@ -6,17 +6,24 @@ NODES = 4
 DISKS = 8
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "chef/centos-7.1"
```

Comment:
so do you still need to make changes to the ansible script to make the correct configuration inside the VM?